pkg/services/helpers/neofs/neofs.go

```diff
@@ -223,15 +223,15 @@ func getHash(ctx context.Context, s user.Signer, c Client, addr *oid.Address, ha
 }
 
 func parseRange(s string) (*object.Range, error) {
-	sepIndex := strings.IndexByte(s, rangeSep)
-	if sepIndex < 0 {
+	before, after, ok := strings.Cut(s, "|")
```

Comment:
1. `rangeSep` should still be used as a constant.
2. Semantic names can be used for variables, like `offsetPart, lengthPart, ok := ...`